docs/zh/latest/plugins/request-id.md

```diff
@@ -71,6 +72,99 @@ X-Request-Id: fe32076a-d0a5-49a6-a361-6c244c1df956
 ......
 ```
 
+### 使用 snowflake 算法
```

Comment:
Let's update the English doc too. We can use an online translator for help.

Reply:
First confirm that the document is ok, I am translating to English

Reply:
OK

Reply:
The English document has been added